pkg/plugins/golang/v4/scaffolds/internal/templates/dockerfile.go

```diff
@@ -61,7 +61,7 @@ COPY internal/ internal/
 # was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
```

Comment:
Also, see that the changes here broke the tests

```
      #15 [builder 9/9] RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -race -a -o manager cmd/main.go
      #15 0.430 go: -race requires cgo; enable cgo by setting CGO_ENABLED=1
      #15 ERROR: process "/bin/sh -c CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -race -a -o manager cmd/main.go" did not complete successfully: exit code: 2
      ------
```

It does not seems to work.
So, can you please check out?

Reply:
Will check this. Seems like CGO must be enabled for go build. Will try to leave this part out and start on the other problem where a data race was detected.